src/v/cluster/metadata_cache.cc

```diff
@@ -160,6 +160,10 @@ ss::future<std::vector<node_metadata>> metadata_cache::alive_nodes() const {
     co_return !brokers.empty() ? brokers : _members_table.local().node_list();
 }
 
+std::vector<node_metadata> metadata_cache::all_nodes() const {
```

Comment:
Does this need to copy the vector, or could it return a const ref?

Reply:
_members_table.local().node_list(); returns vector without ref